tigerpath/majors_and_certificates/majors/COM_2018.json

```
@@ -0,0 +1,263 @@
+{
+  "type": "Major",
+  "name": "Comparative Literature",
+  "code": "COM",
+  "degree": "AB",
+  "year": 2018,
+  "urls": [
+    "https://ua.princeton.edu/academic-units/department-comparative-literature",
+    "https://complit.princeton.edu/undergraduate-program/current-majors/requirements"
+  ],
+  "contacts": [
+    {
+      "type": "Chair",
+      "name": "Thomas W. Hare",
+      "email": "thare@princeton.edu"
+    },
+    {
+      "type": "Director of Undergraduate Studies",
+      "name": "Wendy L. Belcher",
+      "email": "wbelcherta@princeton.edu"
+    }
+  ],
+  "req_list": [
+    {
+      "name": "Prerequisites",
+      "max_counted": 1,
+      "min_needed": 1,
+      "explanation": "Foreign Language Requirement. To enter the department, students must be sufficiently knowledgeable in one language other than English to take an upper-level course in it in his or her junior year.\n\nPlan to read a second foreign language before graduation. Proficiency in only one non-English language is required for admission to the department. However, students who concentrate in comparative literature are also expected to study at least one other non-English language and to be able to read in the language by the time they graduate. Such language study may take place before or during their years as departmental concentrators. Some students demonstrate their competency by taking an upper-level course in the literature of that language. Other students gain this competency by taking three terms of language study at Princeton, or two terms and an intensive language course in the summer, or (especially in the case of languages that are no longer spoken) an intensive language course in the summer. A few take a foreign language test administered by the relevant department during the summer.",
+      "course_list": [
+        "LANG 107"
+      ]

```

As with the language requirement here:
https://github.com/TigerPathApp/tigerpath/blob/69937216a9b9c952920647a0d38307b8ebfd98dd/tigerpath/majors_and_certificates/degrees/AB_2018.json#L49-L56
completing a language requirement means taking anything 107 _or higher_.

```suggestion
      "course_list": [
        "LANG 107",
        "LANG 108",
        "LANG 2**",
        "LANG 3**",
        "LANG 4**",
        "LANG 5**"
      ]
```